### DISCUSSION TITLE
Request: Microsoft Teams通知

### DISCUSSION THREAD

**Discussion by uniocto** [growilabs/growi#7726] [Ideas / Requests]
## Informations
MSのTeamsをチャットツールとして利用しております。
slackと同じような感じで、GROWIからTeamsへの通知機能が欲しいです。

**Comment by yuki-takei** [comment]
本 discussion と同内容のリクエストが [#6195](https://github.com/growilabs/growi/discussions/6195) として別途起票されていましたが、#7726 の方がリアクションが多いため #6195 をクローズしました。
